packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -258,6 +258,11 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
           ['@supports (-webkit-overflow-scrolling: touch)']: {
             maxHeight: windowHeight
           },
+          '@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none)': {
```

Comment:
wait... I am confused on the part with `(-ms-high-contrast: none)`... does this mean the border will be both in HC and not in HC modes? And why we are not using the existing HC selector like here:
https://github.com/OfficeDev/office-ui-fabric-react/blob/7b108c2f0942dafa3193431ec631b66b433c2137/packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.styles.ts#L67

Reply:
Good question. I copied the change from this [commit](https://github.com/OfficeDev/office-ui-fabric-react/pull/7997/files#diff-f9b63ae6ae69f8d6ab55f2f3475b5a52L173). The border doesn't appear in high contrast mode.

Reply:
You should probably use the `[HighContrastSelector]` here as per [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-ms-high-contrast#Values) the `none` is no longer supported. The `HighContrastSelector` is giving you the first part https://github.com/OfficeDev/office-ui-fabric-react/blob/7b108c2f0942dafa3193431ec631b66b433c2137/packages/styling/src/styles/CommonStyles.ts#L1

Reply:
Updated to use `[HighContrastSelector`.